### ISSUE TITLE
Replace Angular icons with TS and JS by default

### ISSUE THREAD

**Issue by moraisbrian** [miguelsolorio/vscode-symbols#226]
Would it be possible to define the pattern of the *.service.ts, *.service.js, *.component.ts and *.component.js files for the typescript and javascript icons?
So that if the user prefers to use Angular or even Nest icons, he does the mapping himself.

Example: 
![image](https://github.com/user-attachments/assets/57b87782-33e2-419b-bf39-f529b08ccc03)

At the moment:
![image](https://github.com/user-attachments/assets/1da92d7d-7854-45a4-b6c0-d7adb785115f)

**Comment by miguelsolorio** [comment]
Yes, you can set the following in your user settings ([ref](https://github.com/miguelsolorio/vscode-symbols?tab=readme-ov-file#files)):

```json
"symbols.files.associations": {
    "*.component.js": "js",
    "*.component.ts": "ts",
    "*.service.js": "js",
    "*.service.ts": "ts"
}
```

But I'll leave this issue open for those that may want to vote to change it. I could see an argument for both sides.

**Comment by miguelsolorio** [comment]
It's been a few years with little feedback on this issue, I'll be closing this issue soon if there isn't additional feedback.
